src/Raven.Studio/typescript/viewmodels/database/status/ongoingTasksStats.ts

```diff
@@ -1339,22 +1724,61 @@ class ongoingTasksStats extends viewModelBase {
         const currentDatum = this.tooltip.datum();
 
         if (currentDatum !== element) {
-            const tooltipHtml = "Gap start time: " + (element).start.toLocaleTimeString() +
-                "<br/>Gap duration: " + generalUtils.formatMillis((element).durationInMillis);
+            const tooltipHtml = "Gap start time: " + element.start.toLocaleTimeString() +
+                "<br/>Gap duration: " + generalUtils.formatMillis(element.durationInMillis);
             this.handleTooltip(element, x, y, tooltipHtml);
         }
     }
 
+    private handleSubscriptionPendingTooltip(itemInfo: subscriptionPendingItemInfo, x: number, y: number) {
+        const currentDatum = this.tooltip.datum();
+
+        if (currentDatum !== itemInfo) {
+            let tooltipHtml = `<strong>*** ${itemInfo.title} ***</strong> <br/>`;
+            tooltipHtml += `Duration: ${generalUtils.formatMillis(itemInfo.duration)} <br/>`;
+            tooltipHtml += `<br>Client URI: <strong>${itemInfo.clientUri}</strong> <br/>`;
```

Comment:
double br between duration and client uri ?

Reply:
done